libs/dh/shared/navigation/src/navigation.service.ts

```diff
@@ -55,23 +55,26 @@ export class DhNavigationService {
       });
   }
 
-  navigate(path: 'details' | 'edit' | 'list', id?: string | number) {
+  navigate(path: 'details' | 'edit' | 'list' | 'back', id?: string | number) {
```

Comment:
nitpick: `back` not used

```suggestion
  navigate(path: 'details' | 'edit' | 'list', id?: string | number) {
```